lbrynet/lbrynet_daemon/LBRYDaemonServer.py

```diff
@@ -29,6 +33,184 @@
 log.addHandler(handler)
 log.setLevel(logging.INFO)
 
+class LBRYDaemonRequest(server.Request):
```

Comment:
I lean toward the "[packages as units of reuse](http://stackoverflow.com/a/107836)" model, and this is unlikely to get reused anywhere else. It includes a slightly ugly hack that only applies to this exact situation (i.e. not deleting the temp files). You're right, though, the file is just too big. I'll move it out.